gui/mod-manager.lua

```diff
@@ -401,6 +402,120 @@ function ModmanageScreen:init()
     }
 end
 
+ModlistMenu = defclass(ModlistMenu, widgets.Window)
+ModlistMenu.ATTRS {
+    view_id = "modlist_menu",
```

Comment:
```suggestion
```

Reply:
you don't need to add `view_id` attributes unless you need to access them by name from another context